### PR TITLE
Fix 404 and fix files names.

### PR DESCRIPTION
**Pull request by Pseudolukian**
### Changelog entry 
* Fix 404 on some links.
* Rename export_import  to export-import in reference/ydb-cli/

### Changelog category 

* Documentation

### CODE REVIEW

**Comment by Pseudolukian** [comment]
https://ydb.tech/docs/en/concepts/datamodel/secrets | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/datamodel/external_table | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/datamodel/external_data_source | Error 404 | Add EN translation moke and add to TOC | OK

https://ydb.tech/docs/en/concepts/federated_query/                          | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/postgresql                | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/clickhouse                | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/s3/external_data_source   | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/s3/external_table         | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/s3/write_data             | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/s3/formats                | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/s3/partitioning           | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/concepts/federated_query/s3/partition_projection   | Error 404 | Add EN translation moke and add to TOC | OK

https://ydb.tech/docs/en/operations/query_plans_optimization                | Error 404 | Add EN translation moke and add to TOC | OK

https://ydb.tech/docs/en/maintenance/manual/replacing_nodes                 | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/maintenance/manual/config-overview                 | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/maintenance/manual/dynamic-config                  | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/maintenance/manual/dynamic-config-selectors        | Error 404 | Add EN translation moke and add to TOC | OK
https://ydb.tech/docs/en/maintenance/manual/dynamic-config-volatile-config  | Error 404 | Add EN translation moke and add to TOC | OK

https://ydb.tech/docs/en/reference/ydb-cli/export-import/                   | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/file-structure     | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/tools-dump         | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/tools-restore      | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/auth-s3            | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/export-s3          | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/import-s3          | Error 404 | Fix file name mistake and add to TOC | OK
https://ydb.tech/docs/en/reference/ydb-cli/export-import/import-file        | Error 404 | Fix file name mistake and add to TOC | OK

File: ydb/docs/en/core/changelog-cli.md [hunk 1]
```diff
@@ -62,17 +62,17 @@ Released on September 7, 2023. To update to version **2.6.0**, select the [Downl
 
 **Bug fixes:**
 
-* Fixed string loss when loading with the [ydb import file json](reference/ydb-cli/export_import/import-file.md) command.
+* Fixed string loss when loading with the [ydb import file json](reference/ydb-cli/export-import/import-file.md) command.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:65]: redirects.yaml needs to be update with these URL changes

File: ydb/docs/en/core/concepts/federated_query.md [hunk 2]
```diff
@@ -0,0 +1 @@
+{% include [NoTraslationAlert](../_includes/alerts/no_translation.md) %}
```
Comment (blinkov) [inline ydb/docs/en/core/concepts/federated_query.md:1]: it's better to name the include file `no-translation.md` as we're moving in that direction

File: ydb/docs/en/core/changelog-cli.md (hunk 1 above)
Comment (Pseudolukian) [reply ydb/docs/en/core/changelog-cli.md:65]: .../export_import/... нет в редиректах :)

File: ydb/docs/en/core/concepts/federated_query.md (hunk 2 above)
Comment (Pseudolukian) [reply ydb/docs/en/core/concepts/federated_query.md]: Переименовал no_translation.md в no-translation.md

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**